airbyte-ci/connectors/pipelines/pipelines/helpers/cli.py

```diff
@@ -10,66 +10,60 @@
 import asyncer
 from jinja2 import Template
-from pipelines.models.steps import CommandResult, StepStatus
+from pipelines.models.steps import CommandResult
 
 ALL_RESULTS_KEY = "_run_all_results"
 
 SUMMARY_TEMPLATE_STR = """
-
+{% if command_results %}
 Summary of commands results
 ========================
-{% for command_name, success in results %}
-{{ '✅' if  success else '❌' }} {{ command_prefix }} {{ command_name }}
+{% for command_result in command_results %}
+{{ '✅' if  command_result.success else '❌' }} {{ command_result.command.name }}
 {% endfor %}
+{% endif %}
 """
 
 DETAILS_TEMPLATE_STR = """
-
-Detailed Errors for failed commands
+{% for command_result in command_results %}
+{% if command_result.stdout or command_result.stderr %}
 =================================
-{% for command_name, error in failed_commands_details %}
-❌ {{ command_prefix }} {{ command_name }}
 
-Error: {{ error }}
+Details for {{ command_result.command.name }} command
+{% if command_result.stdout %}
+STDOUT:
+{{ command_result.stdout }}
+{% endif %}
+{% if command_result.stderr %}
+STDERR:
+{{ command_result.stderr }}
+{% endif %}
+{% endif %}
 {% endfor %}
-=================================
 
 """
 
 
 @dataclass
 class LogOptions:
-    list_errors: bool = False
+    quiet: bool = True
     help_message: str = None
 
 
 def log_command_results(ctx: click.Context, command_results: List[CommandResult], logger: Logger, options: LogOptions = LogOptions()):
     """
     Log the output of the subcommands run by `run_all_subcommands`.
     """
-    command_path = ctx.command_path
+
+    if not options.quiet:
+        details_template = Template(DETAILS_TEMPLATE_STR)
+        details_message = details_template.render(command_results=command_results)
+        logger.info(details_message)
 
     summary_template = Template(SUMMARY_TEMPLATE_STR)
-    results = [(r.command.name, r.status is StepStatus.SUCCESS) for r in command_results]
-    summary_message = summary_template.render(results=results, command_prefix=command_path)
+    summary_message = summary_template.render(command_results=command_results)
     logger.info(summary_message)
 
-    result_contains_failures = any([r.status is StepStatus.FAILURE for r in command_results])
-
-    if result_contains_failures:
-        if options.list_errors:
-            failed_commands_details = [
-                (command_result.command.name, command_result.stderr)
-                for command_result in command_results
-                if command_result.status is StepStatus.FAILURE
-            ]
-            if failed_commands_details:
-                details_template = Template(DETAILS_TEMPLATE_STR)
-                details_message = details_template.render(failed_commands_details=failed_commands_details, command_prefix=command_path)
-                logger.info(details_message)
-        else:
-            logger.info(f"Run `{command_path} --list-errors` to see detailed error messages for failed checks.")
-
```

Comment:
I slightly change the logic here for "verbose by default":
By default we always show `stdout` and `stderr` of formatters.

A `--quiet` flag can be passed the the `format` command to not get the execution details.

I find it better as it will be easier to spot in the CI what files are not passing formatting.

Reply:
Stephane will love this :)